documentation/resource_apache2_mod_php.md

```diff
@@ -0,0 +1,18 @@
+# apache2_mod_php
+
+Enables apache2 module `mod_php`.
+
+This resource assumes that php has already been installed with the [`php` cookbook](https://github.com/sous-chefs/php).
+However, this resource will install the specific apache2 php module package if needed, see the `apache_mod_php_package` helper.
```

Comment:
This should be worded differently to reflect that this isn't needed, however if users want to use it with the php cookbook, then can use that cookbook first but set ``install_package`` to ``false.

Reply:
The point of this helper was to install the missing module package for the distros where it is not included in the default PHP packages, not to install PHP. I guess it serves that purpose now anyway so I'll update the wording.

Reply:
Also, the mod_php package on Suse does not install the rest of PHP. Should I update the package helper to include the php package?

Reply:
This should only be worried about packages that require to get apache working with mod_php. It shouldn't worry about any other package since that can be handled with the php cookbook or by other means. Is the php required to get mod_php to work properly on OpenSUSE?

Reply:
Turns out it was something else on SUSE. False alarm.